### PR TITLE
ndots 2 hardcoded

### PR DESCRIPTION
**Pull request by fandujar**
https://pracucci.com/kubernetes-dns-resolution-ndots-options-and-why-it-may-affect-application-performances.html

Due to bad performance of ndots 5, we decided to set ndots 2 in every deploy and app creation

### CODE REVIEW

File: pkg/server/spec/deploy.go [hunk 1]
```diff
@@ -112,6 +122,16 @@ func (b *DeployBuilder) Build() *Deploy {
 			PreStop: &PreStop{DrainTimeoutSeconds: defaultDrainTimeoutSeconds},
 		}
 	}
+
+	if b.d.DNSConfig == nil {
+		b.d.DNSConfig = &DNSConfig{
+			Options: append([]DNSOptions{}, DNSOptions{
+				Name:  "ndots",
+				Value: "2",
```
Comment (drgarcia1986) [inline pkg/server/spec/deploy.go]: what about creating a config (with that values as default) to hold this configuration?

**Comment by drgarcia1986** [comment]
Hi, @fandujar thanks!
Could you update the CHANGELOG with this modification?

File: pkg/server/spec/deploy.go (hunk 1 above)
Comment (fandujar) [reply pkg/server/spec/deploy.go]: @drgarcia1986 that was my first idea, then I kept hard coded 'cause ndots 2 is the new default value on CoreDNS. I will try to create this config.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [next_version] - 2020-02-03
```
Comment (drgarcia1986) [inline CHANGELOG.md]:
```suggestion
## [NEXT_VERSION]
```

File: pkg/server/spec/deploy.go (hunk 1 above)
Comment (drgarcia1986) [reply pkg/server/spec/deploy.go]: You can follow your heart

**Pull request merged**